ci/install_abseil.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    echo "cmake not installed. Exiting.." >&2
+    exit 1
+fi
+BUILD_DIR=/tmp/
+INSTALL_DIR=/usr/local/
+TAG=20210324.0
+pushd $BUILD_DIR
+git clone --depth=1 -b ${TAG} https://github.com/abseil/abseil-cpp.git
+cd abseil-cpp
+mkdir build && pushd build
+cmake -DBUILD_TESTING=OFF -DCMAKE_CXX_STANDARD=11 \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    ..
+make -j $(nproc)
```

Comment:
If you have ninja, see `/tools/setup-ninja.sh`, that'll give you at least x3-x5 times faster build.

Reply:
One interesting thing here is that gRPC / protobuf, for example, should already be installing Abseil ❓ Maybe it's installed private in gRPC / Protobuf.

Reply:
Yes, I was thinking to add this as part of the existing `cmake_otprotocol_test` job. As part of this PR, we are not building otlp exporter using Abseil.